Makefile

```diff
@@ -15,6 +15,10 @@ build:
 clean:
 	cd $(CPP_DIR) && rm -rf build/ | true
 
+.PHONY: grpc
+grpc:
+	python -m grpc_tools.protoc -I grpc/ --python_out=. --grpc_python_out=. grpc/timemachine/parallel/grpc/service.proto
```

Comment:
nit: would it make sense to add an intermediate `%.proto` target to avoid rebuilding? (not sure there's much benefit)

Reply:
I don't think so? And rebuilding produces the exact same code, so it shouldn't cause and trouble with committing modifications

Reply:
Nice! Good idea putting it in the Makefile as opposed to some obscure comment